exonum-java-binding/core/rust/integration_tests/tests/executor.rs

```diff
@@ -47,8 +47,3 @@ fn concurrent_threads() {
     const THREAD_NUM: usize = 8;
     test_concurrent_threads(EXECUTOR.clone(), THREAD_NUM)
 }
-
-#[test]
-fn nested_attach() {
```

Comment:
I think we shall keep it because it verifies the executor implementation allowing recursive `with_attached`, which some implementations might not permit.